pkg/pulumi-docker-buildkit/sdk/v1.1.17/dockerbuildkit/doc.go

```diff
@@ -0,0 +1,3 @@
+// A Pulumi package for building Docker images with Buildkit.
+//
+package dockerbuildkit
```

Comment:
would it be possible to move these into a separate public repository? Even if we just use the commit hash of the repo as the go dependency, we could set up proper versioning later.